scipy/interpolate/tests/test_interpolate.py

```diff
@@ -929,7 +929,7 @@ def test_mod_invalid_method(self):
             Akima1DInterpolator(x, y, method="invalid")  # type: ignore
 
     def test_extrapolate_attr(self):
-        # 
+        #
```

Comment:
?

Reply:
Oops accidental change thanks
```suggestion
        # 
```